packages/backend-defaults/src/entrypoints/cache/CacheManager.test.ts

```diff
@@ -462,6 +469,31 @@ describe('CacheManager store options', () => {
       expect(result).toBe('my-app:testPlugin');
     });
 
+    it('returns pluginId when store options have no keyPrefix (valkey)', () => {
```

Comment:
Instead of copying and pasting the test which allows for them to get out of sync, could you parameterize the tests instead? This also makes it obvious that the tests are identical.

```js
it.each([
  { type: 'redis', suffix: 'namespace', client: { /* client here */ }],
  { type: 'valkey', suffix: 'keyPrefix', client: { /* client here */ }],
])('returns pluginid when store options have no $suffix ($type)', ({ client, type }) =>{
  const storeOptions = {
    client,
    type,
  };
  // ... the rest of the test is identical
});
```

This also applies to the next test you added.

Reply:
I understand. The body of the following test is identical to several tests before it is why it seemed like they could be combined.